runtime/browser-vr-demo/recipes.js

```diff
@@ -0,0 +1,102 @@
+module.exports = [{
```

Comment:
is this file expected to be different than the non VR browser-demo recipes.js? maybe it is possible to reuse the same file, instead of maintaining both?